lib/importers/google/satisfaction/call_record.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+module Importers
+  module Google
+    module Satisfaction
+      class CallRecord
+        SATISFACTION_VALUE = {
+          'Delighted' => 4,
+          'Vey pleased' => 3,
```

Comment:
Presumably this was a typo somewhere, but is this just for a legacy import or are we still expecting data to have this?